build.gradle

```diff
@@ -31,6 +31,7 @@ plugins {
     id 'com.diffplug.gradle.spotless' version '3.27.2'
     id "com.github.ben-manes.versions" version '0.28.0'
     id 'com.github.jk1.dependency-license-report' version '1.2'
+    id "com.google.protobuf" version "0.8.10"
```

Comment:
I would avoid the usage of protobuf in this PR, we will deal with that later. Please also remove the protbuf files in `src/main/proto` for now.